editor/code/src/goals.ts

```diff
@@ -31,12 +31,14 @@ export class InfoPanel {
   }
 
   panelFactory() {
-    let webviewOpts: WebviewOptions = { enableScripts: true };
```

Comment:
This was an intersection type so you couldn't add enableFindWidget here directly. I've inlined it but the type is really `WebviewOptions & WebviewPanelOptions`.

Reply:
You can also just remove the annotation (or add the intersection to the type annotation)